### ISSUE TITLE
String comparison fails with value returned from papi

### ISSUE THREAD

**Issue by wysohn** [TriggerReactor/TriggerReactor#471]
```
var = placeholder.parse(player, "%betonquest_spaceQuestLine:condition.openedChest%")
#MESSAGE var
IF var == "true"
    #MESSAGE "var is true"
ELSE
    #MESSAGE "var is false"
ENDIF
```

**Comment by wysohn** [comment]
![FinishedIsland8-10653-201](https://user-images.githubusercontent.com/8789836/155655878-2472e5f2-c27c-48a1-89f7-4ebc9fed137a.jpg)
^^ It's a text file

**Comment by Ioloolo** [comment]
> ![FinishedIsland8-10653-201](https://user-images.githubusercontent.com/8789836/155655878-2472e5f2-c27c-48a1-89f7-4ebc9fed137a.jpg) ^^ It's a text file

file content
```
//tempvar = placeholder.parse(player, "%betonquest_spaceQuestLine:condition.openedChest%")

//#MESSAGE tempvar

//IF tempvar == "true"
//    #MESSAGE "You already opened this chest!"
//ELSE
//    #MESSAGE "&aYou find an journal along with an unusual item shoved in the back of this chest. Maybe it could come in useful? &7Tip: You hold quest items in your /backpack."
//    #CMDCON "betonquest event $playername spaceQuestLine.give_item_key_finder"
//    #CMDCON "betonquest event $playername spaceQuestLine.give_item_astro_journal"
//    #CMDCON "betonquest tag $playername add spaceQuestLine.openedChest"
//ENDIF

var = placeholder.parse(player, "%betonquest_spaceQuestLine:condition.openedChest%")

#MESSAGE var.getClass()
var = var + "1"

IF var == "true1"
    #MESSAGE "var is true"
ELSE
    #MESSAGE "var is false"
ENDIF
```

**Comment by Sayakie** [comment]
Resolved on 3.3.0.8-beta.
For more details, check out the #473
